cdap-ui/app/tracker/main.js

```diff
@@ -0,0 +1,373 @@
+
+
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+console.time(PKG.name);
+
+angular
+  .module(PKG.name, [
+
+    angular.module(PKG.name+'.features', [
+      PKG.name+'.feature.tracker'
+    ]).name,
+
+    angular.module(PKG.name+'.commons', [
+
+      angular.module(PKG.name+'.services', [
+        PKG.name+'.config',
+        'ngAnimate',
+        'ngSanitize',
+        'ngResource',
+        'ngStorage',
+        'ui.router',
+        'cask-angular-window-manager',
+        'cask-angular-theme',
+        'cask-angular-focus',
+        'ngCookies'
+      ]).name,
+
+      angular.module(PKG.name+'.filters', [
+        PKG.name+'.services',
+        'cask-angular-capitalize'
+      ]).name,
+
+      'cask-angular-sortable',
+      'cask-angular-confirmable',
+      'cask-angular-promptable',
+      'cask-angular-json-edit',
+      'cask-angular-eventpipe',
+      'cask-angular-observable-promise',
+      'cask-angular-socket-datasource',
+      'cask-angular-dispatcher',
+
+      'mgcrea.ngStrap.datepicker',
+      'mgcrea.ngStrap.timepicker',
+
+      'mgcrea.ngStrap.alert',
+
+      'mgcrea.ngStrap.popover',
+      'mgcrea.ngStrap.dropdown',
+      'mgcrea.ngStrap.typeahead',
+      'mgcrea.ngStrap.select',
+      'mgcrea.ngStrap.collapse',
+
+      // 'mgcrea.ngStrap.modal',
+      'ui.bootstrap.modal',
+      'ui.bootstrap',
+
+      'mgcrea.ngStrap.modal',
+
+      'ncy-angular-breadcrumb',
+      'angularMoment',
+      'ui.ace',
+      'gridster',
+      'angular-cron-jobs',
+      'angularjs-dropdown-multiselect',
+      'hc.marked',
+      'ngFileSaver',
+      'infinite-scroll'
```

Comment:
might not come in this PR, but we should be able to limit these dependencies as well right?